lib/renderer.js

```diff
@@ -58,41 +55,34 @@ Renderer.prototype.link = function(href, title, text) {
     return href;
   }
 
-  var out = '<a href="' + href + '"';
+  let out = `<a href="${href}"`;
 
   if (title) {
-    out += ' title="' + title + '"';
+    out += ` title="${title}"`;
   }
 
-  out += '>' + text + '</a>';
+  out += `>${text}</a>`;
   return out;
 };
 
 // Support Basic Description Lists
-Renderer.prototype.paragraph = function(text) {
-  var result = '';
-  var dlTest = /(^|\s)(\S.+)(<br>:(\s+))(\S.+)/;
-
-  var dl
-    = '<dl>'
-      + '<dt>$2</dt>'
-      + '<dd>$5</dd>'
-    + '</dl>';
-
-  if (text.match(dlTest)) {
-    result = text.replace(dlTest, dl);
-  } else {
-    result = '<p>' + text + '</p>\n';
+Renderer.prototype.paragraph = text => {
+  const dlTest = /(?:^|\s)(\S.+)<br>:\s+(\S.+)/;
+
+  const dl = '<dl><dt>$1</dt><dd>$2</dd></dl>';
+
+  if (dlTest.test(text)) {
```

Comment:
What is a `test` property?

Reply:
In Diff, the lines are off, but this is a refactoring of `text.match (dlTest)`.
[The content of this property is described in MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test).
In `String#replace()`, `RegExp#exec()` is internally called. Therefore, I changed it to `RegExp#test()` in order not to run RegExp#exec() twice.

Reply:
OK. I understood :)